stylesheets/style.css

```diff
@@ -5,3 +5,219 @@ main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  line-height: 150%;
+  font-size: 14px;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+h1,
+h2 {
+  font-family: Georgia, "Times New Roman", Times, serif;
+  font-size: 1.75rem;
+}
+
+button {
+  padding: 10px;
+  width: 100%;
+  font-weight: bold;
+  border-radius: 2px;
+}
+
+/* For up to 768px */
+@media (max-width: 768px) {
+  nav {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 70px;
+    background: #fff;
+    width: 100%;
+    padding: 10px 20px;
+    border-bottom: 2px solid #ccc;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  nav img {
+    height: 100%;
+    width: auto;
+  }
+
+  nav ul {
+    display: none;
+    visibility: hidden;
+  }
+
+  header {
+    margin-top: 90px;
+    display: flex;
+    flex-direction: column;
+    padding: 0 20px;
+  }
+
+  header img {
+    width: 100%;
+    height: auto;
+    order: 3;
+  }
+
+  button[name="button"] {
+    border: none;
+    background: #192592;
+    color: #fff;
+    padding: 10px;
+  }
+
+  button + p {
+    text-align: center;
+  }
+
+  main div {
+    text-align: center;
+  }
+
+  main div:first-child {
+    background: #f4f3f4;
+    padding: 40px 20px;
+  }
+
+  main button {
+    background: #fff;
+    border: 1px solid #192592;
+    color: #192592;
+    margin-bottom: 30px;
+  }
+
+  p {
+    margin-bottom: 30px;
+  }
+
+  main img {
+    display: block;
+    padding: 15px;
+    margin: 0 auto 15px auto;
+    width: 50%;
+    height: auto;
+  }
+
+  main div + div {
+    border-bottom: 1px solid #ccc;
+    padding: 20px;
+  }
+
+  main div + div > p:first-child {
+    font-family: Georgia, "Times New Roman", Times, serif;
+    font-size: 1.25rem;
+    font-weight: bold;
+    margin-bottom: 0;
+  }
+
+  footer div {
+    padding: 0 20px;
+  }
+
+  footer div:first-child {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    background: url(/images/iso-slack.png) no-repeat;
+    background-size: 100px 100px;
+    padding: 100px 20px 0 20px;
+  }
+
+  /*footer div:first-child img {
```

Comment:
try not to leave commented code, is a bad habit that can confuse other developers on the project